features/signals_railway_signals.yaml

```diff
@@ -4707,114 +4727,266 @@ features:
       - { tag: 'railway:signal:main:form', value: 'light' }
       - { tag: 'railway:signal:main:substitute_signal', value: 'IT:AVV' }
 
+     - description: Indicatori di partenza (Avvio)
+     country: IT
+     icon: { default: 'it/PAR-AVV' }
+     tags:
+       - { tag: 'railway:signal:departure', value: 'IT:PAR' }
+       - { tag: 'railway:signal:departure:form', value: 'light' }
+       - { tag: 'railway:signal:departure:substitute_signal', value: 'IT:AVV' }
+
+#-------------
+# ... to there. with the same result. ----
+
+  - description: Indicatori di partenza
+    country: IT
+    icon: { default: 'it/PAR' }
+    tags:
+      - { tag: 'railway:signal:departure', value: 'IT:PAR' }
+      - { tag: 'railway:signal:departure:form', value: 'light' }
+
   - description: 1ª categoria (1 light)
     country: IT
     icon:
       match: 'railway:signal:main:states'
       cases:
+        - { any: ['(Y)','Y'], value: 'it/main-1v-Y' }
         - { exact: 'G', value: 'it/main-1v-G' }
       default: 'it/main-1v-R'
     tags:
       - { tag: 'railway:signal:main', value: 'IT:1V' }
       - { tag: 'railway:signal:main:form', value: 'light' }
 
+  - description: 1ª categoria quadro (1 light)
+    country: IT
+    icon:
+      match: 'railway:signal:main:states'
+      cases:
+        - { any: ['(Y)','Y'], value: 'it/main-s-1v-Y' }
+        - { exact: 'G', value: 'it/main-s-1v-G' }
+      default: 'it/main-s-1v-R'
+    tags:
+      - { tag: 'railway:signal:main', value: 'IT:1V' }
+      - { tag: 'railway:signal:main:form', value: 'light' }
+      - { tag: 'railway:signal:main:shape', value: 'square' }
```

Comment:
We have to add the matching tags `railway:signal:main:shape` (and similar tags below) to the list of tags at the top of the file.